Dockerfile.cirun

```diff
@@ -1,9 +1,14 @@
 FROM quay-ci-base
+
+ENV ENCRYPTED_ROBOT_TOKEN_MIGRATION_PHASE remove-old-fields
```

Comment:
This should no longer be needed

Reply:
@thomasmckay It's not yet removed from `master`. Should we wait until that change comes through (next week?) to make it?

Reply:
sorry, thought it was in master already. nm this comment